### PR TITLE
Coverity fixes

### PR DESCRIPTION
**Pull request by ariscop**

### CODE REVIEW

File: modules/core/m_ban.c [hunk 1]
```diff
@@ -152,16 +152,6 @@ ms_ban(struct Client *client_p, struct Client *source_p, int parc, const char *p
...
-        rb_free(aconf->user);
-        aconf->user = NULL;
-        rb_free(aconf->host);
-        aconf->host = NULL;
-        operhash_delete(aconf->info.oper);
-        aconf->info.oper = NULL;
-        rb_free(aconf->passwd);
-        aconf->passwd = NULL;
-        rb_free(aconf->spasswd);
-        aconf->spasswd = NULL;
```
Comment (Xe) [inline modules/core/m_ban.c]: what was the reason for this being removed?

Comment (ariscop) [inline modules/core/m_ban.c]: deactivate_conf calls free_conf which already frees those pointers

Comment (Xe) [inline modules/core/m_ban.c]: ah, LGTM then

Comment (ariscop) [inline modules/core/m_ban.c]: doesn't set them null though apparently, though deactivate_conf is called elsewhere so its *probably* not an issue, i'll check

Comment (ariscop) [inline modules/core/m_ban.c]: turns out free_conf frees aconf as well, so this is also a use after free

**Pull request merged**